public/plugins/ckeditor/skins/bootstrapck/scss/components/_mainui.scss

```diff
@@ -0,0 +1,189 @@
+/*
+Copyright (c) 2003-2014, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md or http://ckeditor.com/license
+*/
+
+/*
+mainui.css (part of editor.css)
+=================================
+
+This file styles the basic structure of the CKEditor user interface - the box
+that holds everything.
+
+CKEditor offers two main editing modes. The main UI blocks that compose these
+modes are:
+
+    For "Theme UI" mode, the one most generally used:
+
+    +-- .cke_chrome ----------------------+
+    |+-- .cke_inner ---------------------+|
+    || +-- .cke_top -------------------+ ||
+    || |                               | ||
+    || +-------------------------------+ ||
+    || +-- .cke_contents --------------+ ||
+    || |                               | ||
+    || +-------------------------------+ ||
+    || +-- .cke_bottom ----------------+ ||
+    || |                               | ||
+    || +-------------------------------+ ||
+    |+-----------------------------------+|
+    +-------------------------------------+
+
+    For "Inline Editing" mode:
+
+    +-- .cke_chrome .cke_float------------+
+    |+-- .cke_inner ---------------------+|
+    || +-- .cke_top -------------------+ ||
+    || |                               | ||
+    || +-------------------------------+ ||
+    |+-----------------------------------+|
+    +-------------------------------------+
+
+Special outer level classes used in this file:
+
+    .cke_hc: Available when the editor is rendered on "High Contrast".
+
+*/
+
+/* The outer boundary of the interface. */
+.cke_chrome {
+    /* This is <span>, so transform it into a block.*/
+    display: block;
+    border: 1px solid $hr-border;
+    border-radius: $border-radius;
+    padding: 0 3px;
+    background: $gray-lighter;
+}
+
+/* The inner boundary of the interface. */
+.cke_inner {
+    /* This is <span>, so transform it into a block.*/
+    display: block;
+
+    -webkit-touch-callout: none;
+
+    background: transparent;
+    padding: 0;
+}
+
+/* Added to the outer boundary of the UI when in inline editing,
+   when the UI is floating. */
+.cke_float {
+    /* Make white the space between the outer and the inner borders. */
+    border: none;
+
+    .cke_inner {
```

Comment:
Selector `cke_inner` should be written in hyphenated BEM (Block Element Modifier) format